man/systemd.exec.xml

```diff
@@ -1707,6 +1719,10 @@
                 <entry>any of the above</entry>
                 <entry>any of the above</entry>
               </row>
+
+              <row>
+                <entry namest="results" nameend="code">Note: the process may be also terminated by a signal not sent by systemd. In particular the process may send send an arbitrary signal to itself in a handler for any of the non-maskable signals. Nevertheless, in the <literal>timeout</literal> and <literal>watchdog</literal> rows above only the signals that systemd sends have been included.</entry>
```

Comment:
Imagine that poor little process trying to send signal to itself, over and over and over :)